repos/system_upgrade/common/models/luksdump.py

```diff
@@ -0,0 +1,57 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class LuksToken(Model):
+    """
+    Represents a single token associated with the LUKS device
+    """
+    topic = SystemInfoTopic
+
+    token_id = fields.Integer()
+    """
+    Token ID (as seen in the luksDump)
+    """
+
+    keyslot = fields.Integer()
+    """
+    ID of the associated keyslot
+    """
+
+    token_type = fields.String()
+    """
+    Type of the token. For "clevis" type the concrete subtype (determined using
+    clevis luks list) is appended e.g. clevis-tpm2. clevis-tang, ...
+    """
+
+
+class LuksDump(Model):
```

Comment:
Checking the logic, I suggest to create one msg that will envelope all created LuksDump (list(Model(LuksDump)). It has some benefits:
* you can check that only one msg exists at all (e.g. no other actors, like custom/third-party, are produced the msg too)
* kind of easier debugging when checking leapp.db, as you find the msg and just go through the data; however for multiple msgs it's kind of more noise - you have more reading actually

It's not blocker, but I suggest to consider the change. You can find both principles used in the project, so no blocker at all. But from my experience having just one msg for "system facts" is little bit more comfortable. Keeping it on you what you is your preference.